### PR TITLE
Area 51 Event

### PR DESCRIPTION
**Pull request by starspritechippy**
I hope this covers everything and I didn't forget something.
There's a new converter to include adventure number 51, there are some special items, and it's basically impossible to do, or at least really low percentage. I don't feel comfortable touching the adventure success chances, but I feel that this being almost impossible to beat is also sort of realistic?
If you want to do something to increase survival chance for adventure 51, that would be cool too.
Here's a list of changes that *you* need to do, since I can't edit these: [Click!](https://mystb.in/gacoqecile.cs)

If there's anything major I forgot, then I'm really sorry 😬

### CODE REVIEW

File: cogs/adventure/__init__.py [hunk 1]
```diff
@@ -88,14 +88,15 @@ def __init__(self, bot):
...
     @commands.command(aliases=["mission", "a", "dungeon"])
     @locale_doc
-    async def adventure(self, ctx, dungeonnumber: IntFromTo(1, 20)):
+    async def adventure(self, ctx, dungeonnumber: AreaFiftyOneInt(1, 20)):
         _("""Sends your character on an adventure.""")
         if dungeonnumber > int(rpgtools.xptolevel(ctx.character_data["xp"])):
-            return await ctx.send(
-                _("You must be on level **{level}** to do this adventure.").format(
-                    level=dungeonnumber
+            if dungeonnumber != 21: # bybass level req. for the event
```
Comment (Gelbpunkt) [inline cogs/adventure/__init__.py]: Typo here

**Review by Gelbpunkt — APPROVED** [review]
Looks good to me apart from that typo

**Comment by Gelbpunkt** [comment]
Also can you please update config.example.py with your proposed changes?

**Comment by starspritechippy** [comment]
Will do!

**Comment by starspritechippy** [comment]
Hope there are no unexpected side effects 😬 
If you find anything else, I'll get to it tomorrow.

**Pull request merged**